### PR TITLE
update dialects up-to-20220216

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: message_definitions/v1.0/AVSSUAS.xml [hunk 1]
```diff
@@ -3,16 +3,16 @@
 <!-- AVSS is first commercially available products are parachute recovery systems for commercial drones. -->
 <!-- AVSS contact info: -->
 <!-- company URL: https://www.avss.co -->
-<!-- email contact: josh.boudreau@avss.co thomas.li@avss.co-->
+<!-- email contact: josh.boudreau@avss.co thomas.li@avss.co llxxgg@gmail.com-->
 <!-- mavlink messenger ID range: 60050 - 60099,  mavlink command ID range: 60050 - 60099-->
 <mavlink>
-  <include>common.xml</include>
-  <version>1</version>
+  <include>../../message_definitions/v1.0/common.xml</include>
```
Comment (tridge) [inline message_definitions/v1.0/AVSSUAS.xml:9]: if this has since been reverted we shouldn't apply it

**Review by tridge — CHANGES_REQUESTED** [review]

**Comment by peterbarker** [comment]
I've made some of these changes in https://github.com/ArduPilot/mavlink/pull/270

I'll need to consider minimal.xml and development.xml changes separately.

The uAvionix messages in our repo should probably go upstream.

**Comment by amilcarlucas** [comment]
I will rebase this soonish.

File: message_definitions/v1.0/minimal.xml [hunk 2]
```diff
@@ -593,6 +620,12 @@
       <entry value="175" name="MAV_COMP_ID_GIMBAL6">
         <description>Gimbal #6.</description>
       </entry>
+      <entry value="180" name="MAV_COMP_ID_BATTERY">
+        <description>Battery #1.</description>
+      </entry>
+      <entry value="181" name="MAV_COMP_ID_BATTERY2">
```
Comment (peterbarker) [inline message_definitions/v1.0/minimal.xml:626]: I really don't get why we're bothering to name components.  There are just too great a variety/count of devices out there....

e.g. ArduPilot supports 9 batteries, each of which could be mavlink, and each of which ArduPilot could theoretically handle messages for...

**Review by peterbarker — APPROVED** [review]
LGTM

**Pull request merged**